README.md

```diff
@@ -36,79 +36,82 @@ helm repo update
 
 1. Clone the latest version of gateway-certs-generator from  the `master` branch.
 
-```console
-git clone https://github.com/kubeslice/gateway-certs-generator.git
-cd gateway-certs-generator
-```
+   ```console
+   git clone https://github.com/kubeslice/gateway-certs-generator.git
+   cd gateway-certs-generator
+   ```
 
 2. Adjust image name variable `IMG` in the [`Makefile`](Makefile) to change the docker tag to be built.
    Default image is set as `IMG ?= aveshasystems/gateway-certs-generator:latest`. Modify this if required.
 
-```console
-make docker-build
-```
-### Running Local Image on Kind Clusters
+   ```console
+   make docker-build
+   ```
+### Run Local Image on Kind Clusters
 
-1. Loading gateway-certs-generator Image into your kind cluster ([kind](https://kind.sigs.k8s.io/docs/user/quick-start/#loading-an-image-into-your-cluster)).
-   If needed, replace `aveshasystems/gateway-certs-generator` with your locally built image name in the previous step.
+1. Load the gateway-certs-generator image into your kind cluster ([kind](https://kind.sigs.k8s.io/docs/user/quick-start/#loading-an-image-into-your-cluster)).
+   If needed, modify `aveshasystems/gateway-certs-generator` with your locally built image name in the previous step.
    
 * Note: If you use a named cluster, you must specify the name of the cluster you wish to load the images into. See [loading an image into your kind cluster](https://kind.sigs.k8s.io/docs/user/quick-start/#loading-an-image-into-your-cluster).
-```console
-kind load docker-image aveshasystems/gateway-certs-generator --name cluster-name
-```
-Example
-```console
-kind load docker-image aveshasystems/kubeslice-controller --name kind
-```
-
-2. Check the loaded image in the cluster. Modify node name if required.
-
-* Note: `kind-control-plane` is the name of the Docker container. Modify if needed.
-```console
-docker exec -it kind-control-plane crictl images
-```
-
-### Deploying in a Cluster
-1. Create a chart values called `yourvaluesfile.yaml` file. Refer to [values.yaml](https://github.com/kubeslice/charts/blob/master/charts/kubeslice-controller/values.yaml) on how to adjust this and update the `kubeslice-controller` image to the local build image.
+  
+   ```console
+   kind load docker-image aveshasystems/gateway-certs-generator --name cluster-name
+   ```
+   Example
+   ```console
+   kind load docker-image aveshasystems/kubeslice-controller --name kind
+   ```
+
+2. Check the loaded image in the cluster. Modify the node name if required.
+
+* Note: `kind-control-plane` is the name of the Docker container. Modify if needed. 
+  
+  ```console
+  docker exec -it kind-control-plane crictl images
+  ```
+
+### Deploy on a Cluster
+1. Create a chart values file `yourvaluesfile.yaml` file. Refer to [values.yaml](https://github.com/kubeslice/charts/blob/master/charts/kubeslice-controller/values.yaml) to update the `kubeslice-controller` image to the local build image.
```

Comment:
chart values file called `yourvalues...` 

delete the file at the end of this sentence. The sentence has 2 instances of `files`.